### PR TITLE
chore: investigate stackblur module update

### PR DESCRIPTION
**Pull request by mhatvan**
## Description

`stackblur` API usage changed in https://github.com/esimov/stackblur-go/commit/964742d57924526c65efa9d67afa5a929cbcda8c, I updated the code according to the new API.

I was not able to test this properly with the tool running due to multiple issues:
- I receive `undeclared name: MustAsset` in `output.go`, but was able to workaround it with `github.com/andboson/ab-go/bindata` and then using `bindata.MustAsset`, how is this supposed to work usually?
- Both the tests and a dry run of GitHub actions with https://github.com/nektos/act failed me with `asset: Asset(Hack-Regular.ttf): Asset Hack-Regular.ttf not found`, how is that supposed to work?
- I tried executing `make internal/img/font-hack.go` but it fails as well, that file does not even exist in the repository?

Note that this is my first experience with go, so go easy on me ;)

## Changes 
- update `stackblur` to v1.0.2
- replace `stackblur.Process()` with `stackblur.Run()`

Closes https://github.com/homeport/termshot/issues/38

### CODE REVIEW

**Comment by HeavyWombat** [comment]
Danke! This is awesome. Will have a look into it. The issue with the font is that I decided to not store the font itself in this repository. Therefore, it needs to be downloaded for each build to be packed into the binary. This is why `internal/img/font-hack.go` is not in the repo, it should be created on-the-fly during the build. I think my rational was that I wanted to avoid to have a binary blob in the repo. Downside is a bit more non standard build. I want to rewrite that anyway, since `go-bindata` is not required anymore with Go 1.17 having the embedding feature.

**Comment by mhatvan** [comment]
Sehr gerne! Yeah, I assumed you wanted to do some steps on the fly, just wasn't able to get it to work then. 
A `CONTRIBUTING.md` file is always helpful in these cases.

I am curious how you will finish up this PR. It was great to help out and take some first steps in go at the same time ;)

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ tmp
 dist
 internal/img/font-*.go
 **/*.coverprofile
+cmd/termshot
```
[the remark this answers is not in the document]
Comment (mhatvan) [reply .gitignore:5]: I am using Linux Manjaro and on `go build`, it outputs a `termshot` binary that I wanted to ignore. 
I verified locally before that code changes in `main.go` are still tracked.

We could also update it to `cmd/termshot/termshot` instead if you prefer?

[the remark this answers is not in the document]
Comment (HeavyWombat) [reply .gitignore:5]: Ok, I see. I would like to tackle this independently of this PR here. Like I said, I will rework the build process anyway. Can you remove the added line? Then we can proceed to merge this PR.

**Review by HeavyWombat — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (mhatvan) [reply .gitignore]: `go build ./..` doesn't seem to work (with `oh-my-zsh` at least). It tries to go to the parent directory and then execute, throwing an error about non-existent go files in that dir.

Anyhow, I removed the git ignored code again, ready to merge! :D

**Review by HeavyWombat — APPROVED** [review]

**Pull request merged**